src/js/page_sections/Header.js

```diff
@@ -33,7 +33,7 @@ class Header extends Component {
                 MENU
               </span>
               <span className="coa-text-spacer--vertical"></span>
-              <Link to={'/'}>AUSTIN.GOV</Link>
+              <Link to={'/'}>alpha.AUSTIN.GOV</Link>
```

Comment:
Really minor, but any reason for the curly braces around the to property since this is static and not a variable? Same for the search link.

Reply:
no reason